docs/index.bs

```diff
@@ -3025,11 +3025,12 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
       <li>Let <var>httpsState</var> be "<code>none</code>".</li>
       <li>Let <var>referrerPolicy</var> be the empty string.</li>
       <li>Switching on <var>job</var>'s <a>worker type</a>, run these substeps with the following options:
+        <!-- TODO: reorganize algorithm so that the worker environment is created before fetching happens -->
         <dl>
         <dt><em>"<code>classic</code>"</em></dt>
-        <dd><p><a>Fetch a classic worker script</a> given <var>job</var>’s <a lt="URL serializer">serialized</a> <a href="#dfn-job-script-url">script url</a>, <var>job</var>’s <a href="#dfn-job-client">client</a>, and "<code>serviceworker</code>".</p></dd>
+        <dd><p><a>Fetch a classic worker script</a> given <var>job</var>’s <a lt="URL serializer">serialized</a> <a href="#dfn-job-script-url">script url</a>, <var>job</var>’s <a href="#dfn-job-client">client</a>, "<code>serviceworker</code>", and the to-be-created <a>environment settings object</a> for this service worker.</p></dd>
```

Comment:
An environment settings object for each service worker execution changes. [Run Service Worker](https://w3c.github.io/ServiceWorker/#run-service-worker-algorithm) step 7 creates one before run the script in step 14. So, I guess we should set the script's settings object slot to the created environment settings object in Run Service Worker. And just pass null to Fetch a classic worker script algorithm's script settings object param here, if that makes sense. For that, additional guards would be needed in _create a classic script_ I guess.